source-browser.html

```diff
@@ -182,6 +193,27 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
       }
 
       $(document).ready(() => {
+        $(window).on('scroll resize load', () => {
```

Comment:
Let's pull the anonymous function here into a function called `updateNavbarVisibility`.

Reply:
No, I didn't mean the whole thing, I meant just the arrow function.

Reply:
Okay, I'll fix that.

Reply:
So just `          TOPICS.forEach(({ name }) => {
            const navLinkPosition = $(`#${name}`).position().top;
            $(`#${name}-nav-link`).text(`${name} ${navLinkPosition > bottom ? '↓' : '↑'}`);
            $(`#${name}-nav-list`).toggle(top >= navLinkPosition || navLinkPosition >= bottom);		
          });`?

Reply:
As I mentioned earlier, please use code blocks via 4 space indents.

    $(window).on('scroll resize load', updateNavbarVisibility)

is what I'm looking for.